aoc/src/main/scala/aoc2022/exercise03.scala

```diff
@@ -0,0 +1,55 @@
+package aoc2022
+
+import aoc2022.exercise03.RuckSack
+import aoc2022.standardAoc.readInput
+
+import scala.io.Source
+
+val letterMapper = ('a' to 'z')
+  .zipWithIndex
+  .map(x => x(0).toString -> x(1))
+  .toMap
+
+def getPoints(letter: String, startVal: Int ): Int =
+  // add the startnumber with the index in the alphabete. Lc starts at 1 and cap at 27
+  startVal + letterMapper.getOrElse(letter.toLowerCase, 0)
+
+
+object exercise03 extends App:
+
+  case class RuckSack(inp: List[String]):
+    // Foldleft of tail that keeps elements that occur in every sublist. Make set to avoid doubles
+    val matchLetter = inp.tail
```

Comment:
note that this can throw an exception as the inp might be empty

```
scala> List().tail
java.lang.UnsupportedOperationException: tail of empty list
  at scala.collection.immutable.Nil$.tail(List.scala:664)
  at scala.collection.immutable.Nil$.tail(List.scala:661)
  ... 34 elided
```